### PR TITLE
Feature/DMG

### PR DESCRIPTION
**Pull request by RandyStevens**
This could be a potential solution to #139 

Uses a FOSS cli node app called create-dmg by @sindresorhus.

### CODE REVIEW

File: create_dmg.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if [[ "$SHOULD_BUILD" == "yes" ]]; then
+  if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+    cd VSCode-darwin
+    create-dmg ../VSCodium-darwin-${LATEST_MS_TAG}.app
```
Comment (RandyStevens) [inline create_dmg.sh]: I tried to follow your lead for this bashfile.

Comment (stripedpajamas) [reply create_dmg.sh]: I think this should be `create-dmg VSCodium.app ..`

The syntax is `create-dmg <src> [dst]` and you want the generated `dmg` file as a sibling to the `zip` file

**Comment by stripedpajamas** [comment]
This looks great @RandyStevens 🙌 🙌 🙌 

I tested it out locally and the generated `dmg` looks good. I think it will be a great addition to the available download options.

File: README.md [hunk 2]
```diff
@@ -93,6 +93,7 @@ Please note that some Visual Studio Code extensions have licenses that restrict
 
 ## <a id="supported-os"></a>Supported OS
 - [x] OSX x64 (zipped app file)
+- [x] OSX x64 (DMG installer)
```
Comment (stripedpajamas) [inline README.md]: Combine these lines to be `OSX x64 (zip, dmg)`

File: create_dmg.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if [[ "$SHOULD_BUILD" == "yes" ]]; then
+  if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+    cd VSCode-darwin
+    create-dmg VSCodium-darwin-${LATEST_MS_TAG}.app ../
```
Comment (RandyStevens) [inline create_dmg.sh]: Here is the command with the correct syntax.

Comment (stripedpajamas) [reply create_dmg.sh]: The app file here actually doesn't have the version, so just `VSCodium.app` (when we make the zip file is when the version gets added to the file name)

Comment (RandyStevens) [reply create_dmg.sh]: Let me know if the version tag there should be dropped.

File: create_dmg.sh [hunk 4]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if [[ "$SHOULD_BUILD" == "yes" ]]; then
+  if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+    cd VSCode-darwin
+    create-dmg VSCodium.app ..
```
Comment (RandyStevens) [inline create_dmg.sh]: Got it this time! 😅Thanks for the help!

**Comment by stripedpajamas** [comment]
LGTM

**Pull request merged**